openlibrary/i18n/zh/messages.po

```diff
@@ -0,0 +1,782 @@
+msgid "Waiting for %d day"
+msgid_plural "Waiting for %d days"
+msgstr[0] "等待 %d 天"
+msgstr[1] "等待 %d 天"
+
+#: diff.html:7
+#, python-format
+msgid "Diff on %s"
+msgstr "不同于 %s"
+
+#: diff.html:26
+msgid "Added"
+msgstr "已添加"
+
+#: diff.html:27
+msgid "Modified"
+msgstr "已修改"
+
+#: diff.html:28
+msgid "Removed"
+msgstr "已移除"
+
+#: diff.html:29
+msgid "Not changed"
+msgstr "没有改变"
+
+#: diff.html:39
+#, python-format
+msgid "Revision %d"
+msgstr "修订 %d"
+
+#: AuthorList.html:9 SearchResultsWork.html:39 books/check.html:32
+#: books/edit/edition.html:75 books/show.html:16 covers/book_cover.html:41
+#: covers/book_cover_single_edition.html:36 covers/book_cover_work.html:32
+#: diff.html:42 lists/preview.html:20 type/edition/publisher_line.html:15
+#: type/edition/view.html:188 type/work/view.html:188
+msgid "by"
+msgstr "出自于"
+
+#: diff.html:44
+msgid "by Anonymous"
+msgstr "作者：匿名"
+
+#: diff.html:110
+msgid "Differences"
+msgstr "不同之处"
+
+#: diff.html:168
+msgid "Both the revisions are identical."
+msgstr "两次修订都是相同的。"
+
+#: edit_yaml.html:14 lib/edit_head.html:17
+msgid "You're in edit mode."
+msgstr "您现在处于编辑模式。"
+
+#: edit_yaml.html:15 lib/edit_head.html:18
+msgid "Cancel, and go back."
+msgstr "取消并返回。"
+
+#: edit_yaml.html:22
+msgid "YAML Representation:"
+msgstr "YAML 代表:"
+
+#: BookPreview.html:11 editpage.html:15
+msgid "Preview"
+msgstr "预览"
+
+#: history.html:23
+msgid "History of edits to"
+msgstr "编辑历史"
+
+#: history.html:31
+msgid "Revision"
+msgstr "修订"
+
+#: RecentChanges.html:17 RecentChangesAdmin.html:20 RecentChangesUsers.html:20
+#: admin/ip/view.html:44 admin/people/edits.html:41 history.html:32
+#: recentchanges/render.html:30 recentchanges/updated_records.html:28
+msgid "When"
+msgstr "何时"
+
+#: RecentChanges.html:19 admin/ip/view.html:46 admin/loans_table.html:46
+#: admin/people/edits.html:43 admin/waitinglists.html:28 history.html:33
+#: recentchanges/render.html:32
+msgid "Who"
+msgstr "何人"
+
+#: RecentChanges.html:20 RecentChangesUsers.html:22 admin/ip/view.html:47
+#: admin/people/edits.html:44 history.html:34 recentchanges/render.html:33
+#: recentchanges/updated_records.html:30
+msgid "Comment"
+msgstr "评论"
+
+#: history.html:35
+msgid "Diff"
+msgstr "差异"
+
+#: history.html:42 history.html:73
+msgid "Compare"
+msgstr "对比"
+
+#: history.html:42 history.html:73
+msgid "See Diff"
+msgstr "查看差异"
+
+#: history.html:48 lib/history.html:78
+#, python-format
+msgid "View revision %s"
+msgstr "查看修订 %s"
+
+#: history.html:85
+msgid "Back"
+msgstr "返回"
+
+#: Pager.html:34 history.html:87 lib/pagination.html:37 showmarc.html:54
+msgid "Next"
+msgstr "下一个"
+
+#: internalerror.html:14
+msgid "Sorry. There seems to be a problem with what you were just looking at."
+msgstr "十分抱歉。您刚刚看的东西似乎出现了问题。"
+
+#: internalerror.html:15
+#, python-format
+msgid ""
+"We've noted the error %s and will look into it as soon as possible. Head for "
+"<a href=\"/\">home</a>?"
+msgstr ""
+"我们已注意到此错误 %s 并将尽快调查此错误。前往 "
+"<a href=\"/\">首页</a>?"
```

Comment:
Q: @cdrini there are a few cases where the msgstr is on a different line than the translations -- does this concatenate correctly?

Reply:
That should be fine; the validation step will confirm for certain though